cedar-policy-validator/overflow-checking-example/policies.txt

```diff
@@ -0,0 +1,34 @@
+// Note: 2^63 == 9223372036854775808; Long can be [-2^63, 2^63 - 1]
+// 1 <= principal.x8 <= 8
+
```

Comment:
I'd use constants. Something like `MAX_LONG_MINUS_EIGHT`, `MAX_LONG_MINUS_NINE`, `MIN_LONG_PLUS_EIGHT` and `MAX_LONG_PLUS_NINE`.